aten/src/ATen/Config.h.in

```diff
@@ -11,3 +11,12 @@
 #define AT_NNPACK_ENABLED() @AT_NNPACK_ENABLED@
 #define CAFFE2_STATIC_LINK_CUDA() @CAFFE2_STATIC_LINK_CUDA_INT@
 #define AT_BUILD_WITH_BLAS() @USE_BLAS@
+#ifndef AT_PARALLEL_OPENMP
```

Comment:
Why the ifndef?

Reply:
There are files that include Config.h(maybe transitively) and files that don't. For all of them `AT_PARALLEL_OPENMP` is defined from command line and for those that include Config.h it's redefined in `Config.h`. In the second case warning about redefinition is shown and compilation fails until I guard it with `#ifndef`

Reply:
It seems better to just stop passing `AT_PARALLEL_OPENMP` from command line in this case...

Reply:
@ezyang addressed